ibm/mas_devops/roles/suite_install/templates/core_v1_suite.yml.j2

```diff
@@ -46,3 +46,7 @@ spec:
 {% if mas_img_pull_policy != '' %}
     imagePullPolicy: {{  mas_img_pull_policy  }}
 {% endif %}
+{% if mas_trust_default_cas is defined and mas_trust_default_cas != '' %}
```

Comment:
mas_trust_default_cas has been passed to `bool` it will never equal "" (empty string) .... I think.

IIRC earlier versions of MAS will accept this, so could just add it to all MAS instance settings.  Alternatively to keep with your intent:

`mas_trust_default_cas: "{{ lookup('env', 'MAS_TRUST_DEFAULT_CAS') }}" # don't cast to bool`

This would allow us to see that the var wasn't set, or was set to ""

then we can cast to bool in the template (or do it in elsewhere and save it as `mas_trust_default_cas_bool` or somesuch

Reply:
Good point - I personally prefer only providing the suite flag when the env var is set, which avoids any confusion adding the flag to pre-8.11 Suite CRs. I'll do the boolean cast here in the template.